Dataset.py

```diff
@@ -346,6 +362,9 @@ def _shuffle_frames_in_seqs(self, start, end):
 
   def _load_seqs(self, start, end):
     """
+    TODO: the existance of _load_seqs and load_seqs with the same name is not obvious to me,
+      maybe one should be renamed
```

Comment:
Well, this is somewhat common in Python (and also in general, in any language), that you see some public function, and some internal function (e.g. just recently I saw `GetNext` and `GetNextInternal` in the C++ API of the TF dataset iterator). The public function usually has some simple logic around, and you are supposed to overwrite the internal function.

This specific example mostly follows this common convention. I'm not sure if another name helps. Maybe `_load_seqs_internal`, but you could argue that the `_` prefix already says the same thing.

Of course, I'm always open to extend the documentation to make this more clear.